FREYDAfreeyourdata/Prototype-1.0-BE
# BUG-FIX: Users registration endpoint

#### What does this PR do?
This PR fixes the bug on user registration endpoint

#### Description of Task to be completed?
- fix bug with generating jwt token 

#### How should this be manually tested?
- Use the deployment link below, then navigate to the user registration endpoint
- on successful registration, you should get a token

#### Who should take a look?
@Daltimore
